src/pages/search/project_search/ProjectFacetsFilter.tsx

```diff
@@ -34,13 +35,16 @@ export const ProjectFacetsFilter = ({ projectQuery }: ProjectFacetsFilterProps)
   const selectedParticipants = searchParams.get(ProjectSearchParameter.ParticipantFacet)?.split(',') ?? [];
   const selectedFundingSources = searchParams.get(ProjectSearchParameter.FundingSourceFacet)?.split(',') ?? [];
 
-  const addFacetFilter = (id: string) => {
-    const searchParameters = new URL(id).searchParams;
-    const newSearchParams = new URLSearchParams(
-      `${SearchParam.Type}=${currentSearchType}&${searchParameters.toString()}`
-    );
-    newSearchParams.set(SearchParam.Page, '1');
-    history.push({ search: newSearchParams.toString() });
+  const addFacetFilter = (param: string, key: string) => {
```

Comment:
Updated so that status parameter is not discarded when facets are added. Inspiration taken from result search.